### ISSUE TITLE
Install fails on installing epdlib in venv

### ISSUE THREAD

**Issue by txoof** [txoof/PaperPi#44]
```
      no previously-included directories found matching '.ci'
      adding license file 'LICENSE'
      writing manifest file 'src/Pillow.egg-info/SOURCES.txt'
      warning: build_py: byte-compiling is disabled, skipping.

      running build_ext


      The headers or library files could not be found for jpeg,
      a required dependency when compiling Pillow from source.

      Please see the install instructions at:
         https://pillow.readthedocs.io/en/latest/installation.html

```

**Comment by txoof** [comment]
This looks like Pillow is missing the `libjpeg-dev` package. It can probably be resolved by adding to `./paperpi/debian_packages-paperpi.txt`

**Comment by txoof** [comment]
resolved with [commit](72c42d4993113c2d4dfdd834b58bb7d7202f1e35)
